migrations/migrations/20190628184513-create-user.js

```diff
@@ -25,16 +25,6 @@ module.exports = {
       password: {
         allowNull: false,
         type: Sequelize.STRING
-      },
```

Comment:
Why did you remove these fields?

Reply:
Because I think that is not necessary for the training. Maybe I'm probably wrong.